config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -180,7 +180,7 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: containerd-e2e-ubuntu
-- name: ci-containerd-node-e2e
+- name: ci-containerd-node-e2e-conformance
```

Comment:
let's call them nodeconformance to not mistake with the "real" conformance tests

Reply:
I understand there will be too many "node" there, but keeping only one "node" and adding conformance will be confusing